### ISSUE TITLE
Support for PDV video files

### ISSUE THREAD

**Issue by clarklab** [cadin/panels#21]
I've been kicking the tires and this thing is really neat! Well done. 

A question: is there any way to support video? I've got a little tool that makes PDV files and I think it would open up some really interesting possibilities with video in panels. I looked through the docs but didn't see anything.

Thanks in advance!

**Comment by clarklab** [comment]
Currently toying with a little demo that uses Panels and the crank to advance video frame by frame (kinda like a first-person 3d sim). It's actually a really neat effect, but my process currently is:
- convert video to 12fps
- export image sequence 
- convert images to 1bit
- use [images](https://cadin.github.io/panels/docs/comic-data/layers/#images) to animate sequence on crank

Recently I spotted this little [PDV converter](https://devforum.play.date/t/video-encoder-work-in-progress-mac-only/1390), and it makes some pretty nice video. Curious if I could trigger playback inside a panel, or even better, control playback with the crank.

Of course, I understand this is _beyond_ and niche case, and really I'm just tinkering here, so don't think too much on this one, haha.

**Comment by cadin** [comment]
Yeah, this is probably out of scope for Panels, but you should be able to render the video using a custom function. You could also probably somehow export the video frames as an image sequence and do it that way. Not sure if there's any file size or performance benefits to using the PDV.

I'm going to close this for now, but feel free to add comments here if you make any discoveries or if there's anything I can do to help get you unblocked.
